### PR TITLE
feat: Use the Cayman theme, update for the new API changes, and more

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Use the Cayman theme (modified).
  - Now that GitHub Pages are enabled, we can customize our simple website to make it look better.
- Update for the new API changes in [`gh-actions-html-table-generator`](https://github.com/Panquesito7/gh-actions-html-table-generator/tree/big_update).
- Uses a separate file `profiles.md` for less lag in the main page and `README.md`. IMO, it makes it look better and more professional. The file is fully customizable, thanks to the new API changes. If no file is specified, it will automatically use `README.md`.

**EddieHubCommunity/gh-actions-html-table-generator/pull/25 has to be merged to have this PR working.**
**Wanna see how it looks?** Check out the [website](https://panquesito7.github.io/awesome-github-profiles) and let me know what do you think.

@eddiejaoude, please check both PRs; they are important. Thanks. 🙂

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -4,14 +4,14 @@ jobs:
   table:
     runs-on: ubuntu-latest
     name: Update README from json data
-    if: github.repository_owner == 'EddieHubCommunity'
     steps:
       - uses: actions/checkout@v2
       - name: Read/Write data into README
-        uses: eddiejaoudecommunity/gh-actions-html-table-generator@v0.1
+        uses: Panquesito7/gh-actions-html-table-generator@main
```
Comment (schmelto) [inline .github/workflows/main.yml:10]: I think after the pr merge in gh actions repo this should be changed to eddiehub again

**Comment by AvidCoder101** [comment]
@Panquesito7 I think the website is looking good with a Cayman theme

**Comment by AvidCoder101** [comment]
Also, I completely agree with the `profiles.md`. Thank you for working on this issue!

**Comment by CodingSpecies** [comment]
@Panquesito7 , I like how you made it more clearer, and more efficient by added a separate file for all the files

File: .github/workflows/main.yml (hunk 1 above)
Comment (KukretiShubham) [reply .github/workflows/main.yml:10]: Yep, agree

**Comment by KukretiShubham** [comment]
LGTM

**Review by KukretiShubham — APPROVED** [review]
After changes by @schmelto , LGTM

File: .github/workflows/main.yml (hunk 1 above)
Comment (Panquesito7) [reply .github/workflows/main.yml]: Fixed; thanks for pointing that out. 🙂

**Comment by schmelto** [comment]
- [ ] https://github.com/EddieHubCommunity/gh-actions-html-table-generator/pull/25

**Comment by Panquesito7** [comment]
No response from the author (it's been 20 days) 😅 should we wait for more time or use (for now) this method?

**Review by krishguptadev — CHANGES_REQUESTED** [review]
## Checks Not Passing

Thank you for your pull request. Your contributions are unfortunately not passing our integration checks. Because of this, we are unable to review your pull request.

The errors can be addressed and resolved by:

If you have any questions about resolving this, feel free to ask us and we will gladly work with you.

**Comment by krishguptadev** [comment]
> No response from the author (it's been 20 days) 😅 should we wait for more time or use (for now) this method?

Can we wait like 10 more days 🤔

**Comment by Panquesito7** [comment]
> ## Checks Not Passing
> Thank you for your pull request. Your contributions are unfortunately not passing our integration checks. Because of this, we are unable to review your pull request.
> 
> The errors can be addressed and resolved by:
> 
> If you have any questions about resolving this, feel free to ask us and we will gladly work with you.

This...can't be really helped much. Sometimes the checks fail for no reason. 🤷‍♂️

**Comment by krishguptadev** [comment]
> > ## Checks Not Passing
> > Thank you for your pull request. Your contributions are unfortunately not passing our integration checks. Because of this, we are unable to review your pull request.
> > The errors can be addressed and resolved by:
> > If you have any questions about resolving this, feel free to ask us and we will gladly work with you.
> 
> This...can't be really helped much. Sometimes the checks fail for no reason. 🤷‍♂️

Not sure why is prettier failing: https://github.com/EddieHubCommunity/awesome-github-profiles/runs/4246873146?check_suite_focus=true 🤷‍♂️

**Comment by krishguptadev** [comment]
it says branch not found 🧐

**Pull request merged**